app/helpers/react_components/mentoring/session.rb

```diff
@@ -67,6 +67,7 @@ def mentor_student_relationship
       def links
         {
           mentor_dashboard: Exercism::Routes.mentoring_inbox_path,
+          mentor_queue: Exercism::Routes.mentoring_queue_path,
```

Comment:
I don't know if we want this, or just simply take mentor back to inbox.

Reply:
Back to the queue is good.